.gitmodules

```diff
@@ -13,4 +13,5 @@
 	# url = https://github.com/digital-asset/ghcide.git
 	# url = https://github.com/alanz/ghcide.git
 	# url = https://github.com/wz1000/ghcide.git
-	url = https://github.com/fendor/ghcide.git
+	# url = https://github.com/fendor/ghcide.git
+	url = https://github.com/bubba/ghcide.git
```

Comment:
Is this the correct upstream repo?

Reply:
Until those other PRs and @wz1000s PRs get merged into ghcide, I think so. This checkout is just @fendor's previous submodule + my patches cherry-picked on top